solr/solrj/src/java/org/apache/solr/client/solrj/impl/CloudLegacySolrClient.java

```diff
@@ -87,6 +87,7 @@ protected CloudLegacySolrClient(Builder builder) {
       this.stateProvider = builder.stateProvider;
     }
     this.retryExpiryTime = builder.retryExpiryTime;
+    this.collectionStateCache.timeToLiveMs = builder.timeToLive * 1000L;
```

Comment:
if the builder's timeToLive field were timeToLiveSec then it would be clearer its seconds

Reply:
Done!